### ISSUE TITLE
Loss value doesn't decrease while finetuning 

### ISSUE THREAD

**Issue by SreevaatsavB** [FlagOpen/FlagEmbedding#713]
Tried training the model with the **dense fine-tuning** on this dataset :- [https://huggingface.co/datasets/mteb/scidocs-reranking/viewer/default/validation](url)

With 1 positive and 3 negative sample for each step, and these are the training arguments passed for fine-tuning:- 

`torchrun --nproc_per_node 1 \
-m FlagEmbedding.baai_general_embedding.finetune.run \
--output_dir ./output \
--model_name_or_path BAAI/bge-large-zh-v1.5 \
--train_data /content/train_data.json \
--learning_rate 5e-5 \
--fp16 \
--max_steps 200 \
--per_device_train_batch_size 1 \
--dataloader_drop_last True \
--normlized True \
--temperature 1 \
--query_max_len 80 \
--passage_max_len 280 \
--train_group_size 4 \
--logging_steps 5 \
--save_steps 200 \
--query_instruction_for_retrieval ""`

**The training loss doesn't seem to decrease for this and a few other similar datasets, what could be the reason for this and is there any way to make this model work well?**
Here is the loss fluctuation for reference:- 

<img width="1158" alt="Screenshot 2024-04-22 at 11 00 17 AM" src="https://github.com/FlagOpen/FlagEmbedding/assets/80395869/83644ce5-34a6-4969-80a2-8c2e377fe5ad">

**Comment by SreevaatsavB** [comment]
Thanks for the response, the performance did improve after considering your suggestions.
